<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChumEducation</class>
 <widget class="QWidget" name="ChumEducation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="oceanPhaseLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ocean Phase</string>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>30</y>
     <width>291</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>66</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white
</string>
   </property>
   <property name="text">
    <string>Chum</string>
   </property>
  </widget>
  <widget class="QLabel" name="oceanEducation">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>835</width>
     <height>287</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resources.qrc">:/images/fishEducation/chinookOceanEducation.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="instructionLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>431</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Hover over each image to see more information</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="spawningPhase">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>350</y>
     <width>660</width>
     <height>220</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resources.qrc">:/images/fishEducation/chinookSpawningPhase.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="spawningPhaseLabel">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>520</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Spawning Phase</string>
   </property>
  </widget>
  <widget class="QLabel" name="spawningEducation">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>804</width>
     <height>306</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resources.qrc">:/images/fishEducation/chinookSpawningEducation.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="oceanPhase">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>600</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resources.qrc">:/images/fishEducation/chinookOceanPhase.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>oceanPhase</zorder>
  <zorder>instructionLabel</zorder>
  <zorder>oceanPhaseLabel</zorder>
  <zorder>titleLabel</zorder>
  <zorder>oceanEducation</zorder>
  <zorder>backButton</zorder>
  <zorder>spawningPhase</zorder>
  <zorder>spawningPhaseLabel</zorder>
  <zorder>spawningEducation</zorder>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
